libraries/CurieBLE/src/BLECentralRole.cpp

```diff
@@ -0,0 +1,274 @@
+
```

Comment:
No license in this file either- please add licenses to all the source files that don't already have them